### ISSUE TITLE
Solution seems to Work..

### ISSUE THREAD

**Issue by creepscompilation** [T-vK/ESP32-BLE-Keyboard#169]
RESOLVED:
I took the advice from SunandMittal..

Starting with release v0.3.2
filename = "BleKeyboard.cpp"

Find this line:
pSecurity->setAuthenticationMode(ESP_LE_AUTH_REQ_SC_MITM_BOND);

Replace the above line to the following from release v0.3.0
pSecurity->setAuthenticationMode(ESP_LE_AUTH_BOND);

So, far..
Android connects, re-connects, keyboard codes work
Apple iOS connects, re-connects, keyboard codes work
Windows 10 connects, re-connects, keyboard codes work

**Comment by SunandMittal** [comment]
Dear @creepscompilation, 
good to hear that workaround solution worked for you :-).

regards,
Sunand

**Comment by makerbitter** [comment]
Hello @SunandMittal @creepscompilation I'm using ninBLE option. How could I solve this? I need use nimBLE option activated. I tested in android, connect and reconnect but keyboard codes don't work.

**Comment by SLboat** [comment]
it's work for me too!!!

**Comment by justinkrutz** [comment]
This worked for me as well on my OnePlus 9 with Android 13

**Comment by SirBramble** [comment]
it worked for me too!!!

**Comment by Cengls** [comment]
Holy god Man !!!!!!
WORKING!!!!!!!!!!!!!! 
 IN 
WINDOWS11 
ANDROID10

**Comment by creepscompilation** [comment]
> 

Hey Cengls, 
I'm happy to help..
I wish I could take credit, but I did alot of Google searches until I
was able to find that solution.
Keep bangin' on it..

**Comment by Cengls** [comment]
> Hey Cengls,
> I'm happy to help..
> I wish I could take credit, but I did alot of Google searches until I
> was able to find that solution.
> Keep bangin' on it..


Ur Best ：D

**Comment by MaxLang99** [comment]
YES! This worked for me! Using ESP32C3 to connect to iOS 17. Using as a media controller. Thank you!!!!!
